apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -33,6 +33,24 @@ function getFooter(siteSettings) {
   };
 }
 
+function getMenus(siteSettings) {
```

Comment:
This should be `getNavigation` or `getNavbar` or something along those lines... menus are just part of it. What do we call it in **charterafrica** or **codeforafrica**?